panoptes_aggregation/tests/batch_aggregation/test_batch_aggregation.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from panoptes_aggregation.batch_aggregation import run_aggregation
+
+
+class TestBatchAggregation(unittest.TestCase):
+    def test_save_exports(self):
+        # Test that Panoptes calls are made and files are saved
+        assert 1 == 1
```

Comment:
Will likely want to `patch`/`mock` the Paonoptes class and check the expected `export` calls are made when the function is run.